packages/editor-ui/src/Interface.ts

```diff
@@ -85,6 +86,39 @@ declare global {
 	}
 }
 
+// Based on https://segment.com/docs/connections/spec/identify/#traits
+export type SegmentUserTraits = {
```

Comment:
hmm do we need to specify the whole object? can simplify here to just the ones we are using and custom ones we are adding

Reply:
Nope. For ProfitWell stuff, we only need the `email`. I decided to go with the full in case we end up using other destinations that use other traits, but can also simplify it.

Reply:
hmm the id should be the cloud id though.. that has to be correct